src/ListOfMarginLists.php

```diff
@@ -22,10 +22,10 @@ public function breakTies(TieBreakingMarginComparator $tieBreakingMarginComparat
         $initial = [];
         $result = array_reduce(
             $this->values,
-            function (MarginList $carry, MarginList $current) use ($tieBreakingMarginComparator) {
+            function (array $carry, MarginList $current) use ($tieBreakingMarginComparator) {
                 $currentValues = $current->toArray();
                 usort($currentValues, $tieBreakingMarginComparator);
-                array_push($carry, ...$current);
+                array_push($carry, ...$currentValues);
```

Comment:
Prior to this change, tied Margins were not being sorted, they were simply being returned in the order that they were received. Testing FTW!